blazor/datagrid/style-and-appearance/aggregate.md

```diff
@@ -1,43 +1,50 @@
 ---
 layout: post
-title: Aggregate customization in Blazor DataGrid | Syncfusion
-description: Learn here all about aggregate in Syncfusion Blazor DataGrid and more.
+title: Customize aggregates in Blazor DataGrid | Syncfusion
+description: Learn how to customize aggregate rows in the Syncfusion Blazor DataGrid using CSS, including footer containers and summary cells.
 platform: Blazor
 control: DataGrid
 documentation: ug
 ---
 
-# Aggregate in Syncfusion<sup style="font-size:70%">&reg;</sup> Blazor DataGrid
+# Aggregate customization in Syncfusion Blazor DataGrid
 
-You can customize the appearance of aggregate elements in the Syncfusion<sup style="font-size:70%">&reg;</sup> Blazor DataGrid using CSS. Below are examples of how to customize the aggregate root element and the aggregate cell elements.
+Aggregates render in the DataGrid footer as summary rows and can be styled with CSS. The examples below show how to customize the footer container (aggregate root element) and the individual summary cells. For API details, see the following references:
+- Grid aggregates API: https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridAggregates.html
+- GridAggregateColumn API: https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridAggregateColumn.html
+- AggregateType API: https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.AggregateType.html
+- FooterTemplate API: https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridAggregateColumn.html#Syncfusion_Blazor_Grids_GridAggregateColumn_FooterTemplate
+- AggregateTemplateContext API: https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.AggregateTemplateContext.html
+
+> Note: Ensure GridAggregates are defined so the summary rows render, and that the required theme stylesheet is referenced. When using CSS isolation (.razor.css), target DataGrid internals with the ::deep combinator or wrap the grid in a custom class and scope styles to that wrapper to increase selector specificity. Class names may vary slightly by theme or version; verify in the DOM. Maintain sufficient color contrast for accessibility.
 
 ## Customizing the aggregate root element
 
-To customize the appearance of the Grid's aggregate root elements, you can use the following CSS code:
+To customize the appearance of the DataGrid footer’s aggregate root container, use the following CSS:
```

Comment:
The **e-gridfooter** class is used to style the main container of the aggregate footer row in the DataGrid. To change its appearance, apply CSS like the example below:

Note:
We should highlight the class name, why it was missed here?